stdlib/Profile/src/Profile.jl

```diff
@@ -63,6 +63,27 @@ macro profile(ex)
     end
 end
 
+export @profile_tasks
+
+"""
+    @profile_tasks
```

Comment:
thoughts on name, @NHDaly ? I think you proposed this name but i wonder if you prefer `@profile_walltime` or even a `Wall.@profile` or some such now

(fwiw, i don't have strong feelings here)

Reply:
Yes, I also would prefer to keep it under the same module.

I'm fine with changing it to `@profile_walltime`.